plugins/woocommerce-admin/client/marketplace/components/tabs/tabs.tsx

```diff
@@ -23,11 +24,14 @@ interface Tab {
 	name: string;
 	title: string;
 	href?: string;
+	show_badge?: boolean;
+	badge_text_to_render?: string;
```

Comment:
Do we need two props for this? We could simplify to have one prop: if that is passed, the value of the prop will be rendered.

Reply:
There is a scenario that would compel me to use both props.
[Count in badge](https://github.com/woocommerce/woocommerce/blob/40be78c265dc80f0a7c2623ab221da04cab1bf05/packages/js/components/src/badge/index.tsx#L7) component only accept number type data otherwise we will get this error `Type 'number | undefined' is not assignable to type 'number'`

if we change the badge type to the number and try to render this component this way, it will show the 0 beside the my-subscription tab when the condition is false. 
```
{ tabs[ tabKey ]?.badge && (
						<Badge
							key={ `${ tabKey }-badge` }
							className={ 'woocommerce-badge-color-red' }
							count={  Number(tabs[ tabKey ]?.badge) }
						/>
					) }
```
![Screenshot 2024-04-03 at 1 34 51 PM](https://github.com/woocommerce/woocommerce/assets/10769293/ef9fdff1-8c81-4ab7-8101-e5dcab89f222)